.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: java
+
+dist: trusty
```

Comment:
Trusty is pretty old by now, any reason not to build on xenial?

Reply:
Hey, long time, no see. There's no compelling reason to use Trusty, but there's no advantage to our projects to upgrading to Xenial, and Bionic isn't available on Travis, as far as I'm aware.